content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -12,6 +12,10 @@ By adding a waypoint proxy to your traffic flow you can enable more of [Istio's
 Usage of VirtualService with the ambient data plane mode is considered Alpha. Mixing with Gateway API configuration is not supported, and will lead to undefined behavior.
 {{< /warning >}}
 
+{{< warning >}}
+`EnvoyFilter` is Istio's break glass API for advanced configuration of Envoy proxies. Please note that `EnvoyFilter` is not currently supported for any existing Istio version with waypoint proxies. While it may be possible to use EnvoyFilter in limited use-cases, its use in waypoint proxies is not supported, is discouraged, and may break in future releases as the API evolves. We expect official support will be provided at a later date.
+{{< /warning >}}
```

Comment:
@istio/technical-oversight-committee as discussed in the meeting today.. added to the l7 features ambient doc as well as a note in release notes (above the What's New section).

Reply:
https://deploy-preview-16477--preliminary-istio.netlify.app/latest/docs/ambient/usage/l7-features/ and https://deploy-preview-16477--preliminary-istio.netlify.app/latest/news/releases/1.26.x/announcing-1.26/ show the blurb.